meson.build

```diff
@@ -0,0 +1,76 @@
+project('libebml', 'cpp',
+  version : '1.4.2',
+  default_options : ['warning_level=1', 'cpp_std=c++14'],
+)
+
+ebml_conf = configure_file(output: 'ebml_export.h', configuration: {'EBML_DLL_API': ''})
+libebml_incdir = include_directories(['.'])
+
+install_headers(
+  'ebml/Debug.h',
+  'ebml/EbmlBinary.h',
+  'ebml/EbmlConfig.h',
+  'ebml/EbmlContexts.h',
+  'ebml/EbmlCrc32.h',
+  'ebml/EbmlDate.h',
+  'ebml/EbmlDummy.h',
+  'ebml/EbmlElement.h',
+  'ebml/EbmlEndian.h',
+  'ebml/EbmlFloat.h',
+  'ebml/EbmlHead.h',
+  'ebml/EbmlId.h',
+  'ebml/EbmlMaster.h',
+  'ebml/EbmlSInteger.h',
+  'ebml/EbmlStream.h',
+  'ebml/EbmlString.h',
+  'ebml/EbmlSubHead.h',
+  'ebml/EbmlTypes.h',
+  'ebml/EbmlUInteger.h',
+  'ebml/EbmlUnicodeString.h',
+  'ebml/EbmlVersion.h',
+  'ebml/EbmlVoid.h',
+  'ebml/IOCallback.h',
+  'ebml/MemIOCallback.h',
+  'ebml/MemReadIOCallback.h',
+  'ebml/SafeReadIOCallback.h',
+  'ebml/StdIOCallback.h',
+  ebml_conf,
+  subdir : 'ebml',
+)
+
+#build as static library for simplicity. shared not supported on MSVC.
+libebml = static_library('libebml',
```

Comment:
It turns out this was never fine, even when you submitted this meson.build file to the WrapDB at https://github.com/mesonbuild/wrapdb/pull/236#issuecomment-1003797765 (I didn't merge that, I actually was waiting for you to fix building shared on Linux but someone else merged it anyway, oh well.)

This project does in fact want it to be able to build shared on Linux...

Reply:
>This project does in fact want it to be able to build shared on Linux...

Indeed https://packages.debian.org/bullseye/amd64/libebml5/filelist

Reply:
Switched back to library but do note that shared_libraries do not work at all under MSVC. More info: https://github.com/Matroska-Org/libebml/issues/91